crmsh/ui_cluster.py

```diff
@@ -193,32 +193,22 @@ def looks_like_hostnames(lst):
     ssh         Create SSH keys for passwordless SSH between cluster nodes
     csync2      Configure csync2
     corosync    Configure corosync
-    storage     Partition shared storage (ocfs2 template only)
     sbd         Configure SBD (requires -s <dev>)
     cluster     Bring the cluster online
-    vgfs        Create volume group and filesystem (ocfs2 template only,
```

Comment:
I do have the concern to remove it silently given some other code does use it.

Ps, it makes sense to re-define it to cover XFS/Ext4 on top of HA_LVM as a TODO item.

Reply:
I prefer to keep `vgfs` and add a warning when using `vgfs` to tell this stage already deprecated and will be removed in the future release